src/pages/workspace/tags/WorkspaceViewTagsPage.tsx

```diff
@@ -231,7 +231,9 @@ function WorkspaceViewTagsPage({route}: WorkspaceViewTagsProps) {
             />
         );
     };
-
+    if (!Object.values(currentPolicyTag?.tags ?? {}).some((tag) => tag.enabled)) {
+        Tag.setPolicyTagsRequired(policyID, false, route.params.orderWeight)
```

Comment:
> [!caution]
> There's currently an issue with the implementation which would block this from being merged, basically there's an infinite loop of the `setPolicyTagsRequired` call (see video below) which needs to be fixed such that:
1. The call would only be made **once** when all subtags are **Disabled**.
2. The call would not be made again when the page is refreshed and the tags RHP (right hand panel) is re-opened (like it does in the video towards the end).

https://github.com/user-attachments/assets/2acd046a-4ba1-4a07-8dfc-b043374df6e9

cc @Shahidullah-Muffakir I will be able to pick-up the review and complete the checklist once the issue is fixed and the fix works as expected.

Reply:
Got it. We can update the condition to:

```
    if (!!currentPolicyTag?.required && !Object.values(currentPolicyTag?.tags ?? {}).some((tag) => tag.enabled)) {
        Tag.setPolicyTagsRequired(policyID, false, route.params.orderWeight);
    };
    
```
This will ensure the call is only made when needed and only once. I’ve tested it, and it works as expected. Let me know what you think. Thanks!

Reply:
@ikevin127   I pushed the changes, Please take a look when you can. Thanks!

Reply:
Thanks for addressing the issues, I'll move forward with the review!